src/main/java/mate/jdbc/dao/DriverDaoImpl.java

```diff
@@ -0,0 +1,109 @@
+package mate.jdbc.dao;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.exception.DataProcessingException;
+import mate.jdbc.lib.Dao;
+import mate.jdbc.model.Driver;
+import mate.jdbc.util.ConnectionUtil;
+
+@Dao
+public class DriverDaoImpl implements DriverDao {
+    @Override
+    public Driver create(Driver driver) {
+        String createQuerry = "INSERT INTO drivers (name, license_number) VALUES (?, ?)";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement createStatement
+                        = connection.prepareStatement(createQuerry,
+                        Statement.RETURN_GENERATED_KEYS)) {
+            createStatement.setString(1, driver.getName());
+            createStatement.setString(2, driver.getLicenseNumber());
+            createStatement.executeUpdate();
+            ResultSet generatedKeys = createStatement.getGeneratedKeys();
+            if (generatedKeys.next()) {
+                driver.setId(generatedKeys.getObject(1, Long.class));
+            }
+            return driver;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't add driver to DB", e);
```

Comment:
```suggestion
            throw new DataProcessingException("Can't add driver to DB " + driver, e);
```

lets make exception messages more informative